src/Xamarin.Android.NUnitLite/Xamarin.Android.NUnitLite.csproj

```diff
@@ -353,7 +353,7 @@
       <Name>Mono.Android</Name>
       <Private>False</Private>
     </ProjectReference>
-    <ProjectReference Include="..\Xamarin.Android.Build.Tasks\Xamarin.Android.Build.Tasks.csproj">
+    <ProjectReference Condition=" '$(AndroidApiLevel)' &gt;= 24 " Include="..\Xamarin.Android.Build.Tasks\Xamarin.Android.Build.Tasks.csproj">
```

Comment:
I don't think `>` needs escaping in XML. It *can* be, but I don't believe it's required, and `>=` is certainly easier to read...